src/base/templates/base.hbs

```diff
@@ -127,7 +122,6 @@
   {{else}}
     <script src="/main.bundle.js"></script>
   {{/if}}
-  <script src="{{config.app.api_root}}users/current?format=jsonp&callback=bootstrapCurrentUser"></script>
```

Comment:
This call was causing a race condition, which is now resolved by loading our current user into our redux store, instead of reading it off of the global `Shareabouts.bootstrapped` object